### PR TITLE
Add cilium connectivity test to E2E suite

### PR DESCRIPTION
**Pull request by mcharriere**
Signed-off-by: Matias Charriere <matias@giantswarm.io>

This PR:

- Run `cilium connectivity test`. As a first iteration, it only the `no-policies` and `no-policies-extra` tests.

---

## Checklist

- [ ] I added a CHANGELOG entry
- [ ] I ran E2E tests in the CI pipelines

Add the following comment to trigger the E2E tests:

`/run app-test-suites`

### CODE REVIEW

File: tests/e2e/suites/basic/connectivity_test.go [hunk 1]
```diff
@@ -0,0 +1,104 @@
...
+	"github.com/cilium/cilium/cilium-cli/api"
+	"github.com/cilium/cilium/cilium-cli/connectivity/check"
+	"github.com/cilium/cilium/cilium-cli/defaults"
+	"github.com/cilium/cilium/cilium-cli/k8s"
+	"github.com/cilium/cilium/cilium-cli/sysdump"
+)
+
+func buildTestParams() check.Parameters {
+
+	var params = check.Parameters{
```
Comment (mcharriere) [inline tests/e2e/suites/basic/connectivity_test.go:16]: let's group them into defaults and custom parameters.

**Review by ubergesundheit — APPROVED** [review]
Thanks @mcharriere

**Pull request merged**